requirements.txt

```diff
@@ -1,5 +1,17 @@
-numpy
-netCDF4>=1.4.2
-pygeogrids
-pygeobase>=0.3.18
-pandas!=0.19.0
+# =============================================================================
```

Comment:
These should be repeated in `install_requires` in the `setup.cfg` file. See the instructions created by pyscaffold in this file.